stories/spectra/nasqad.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import { Meta } from '@storybook/react';
+import React, { useState, useEffect } from 'react';
+
+import { Axis, LineSeries, Plot, Heading } from '../../src';
+import type { Series } from '../../src/types';
+import srcData from '../data/nasdaq.json';
+
+export default {
+  title: 'Experimental spectra/Nasqad',
+  argTypes: {
+    refreshFrequency: { defaultValue: 1, control: 'number' },
+    step: { defaultValue: 1, control: 'number' },
+    displayInterval: { defaultValue: 500, control: 'number' },
+  },
+} as Meta;
+
+/** Util functions *****************/
+
+const generateNewXY = (serie: Series, step: number): [number, number] => {
+  const x = serie.x + step;
+  const rand = Math.floor(Math.random() * 10);
+  const y = rand % 2 === 0 ? serie.y + rand * 0.1 : serie.y - rand * 0.1; // generate a new Y by + or - a random value to the last point.y
+  return [x, y];
+};
+
+const getLastData = (
+  data: Array<Series>,
+  displayInterval: number,
+): Array<Series> => {
+  const lastTimestamp = data[data.length - 1].x;
+  const firstIndex = data.findIndex(
+    (serie) => serie.x >= lastTimestamp - displayInterval,
+  );
+  const newVals = data.slice(firstIndex);
+  return newVals;
+};
+
+/** Main Component *****************/
+
+type Props = Record<string, number>;
+export function NasqadExample(props: Props) {
+  const {
+    refreshFrequency, // Page refresher frequency (s)
+    step, // timestamp's step
+    displayInterval, // interval to display (s)
+  } = props;
+
+  function PlotRefresher() {
+    const [data, setData] = useState(srcData);
+
+    useEffect(() => {
+      const timer = setInterval(() => {
+        const [x, y] = generateNewXY(data[data.length - 1], step); // generate a new conrdinates
+        const newData = getLastData([...data, { x, y }], displayInterval); // get last 500s data
+        setData(newData);
+      }, refreshFrequency * 1000);
+      return () => clearInterval(timer);
+    });
```

Comment:
Do you have eslint plugin in vscode ? Normally there should be a warning to enforce the specification of the variables to listen.
So please add `[data]`